src/neo/SmartContract/Native/Oracle/OraclePolicyContract.cs

```diff
@@ -0,0 +1,228 @@
+using Neo.Cryptography.ECC;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Persistence;
+using Neo.SmartContract;
+using Neo.SmartContract.Manifest;
+using Neo.SmartContract.Native;
+using Neo.VM;
+using Neo.VM.Types;
+using System;
+using System.Linq;
+using System.Numerics;
+using Array = Neo.VM.Types.Array;
+
+namespace Neo.Oracle
+{
+    public sealed class OraclePolicyContract : NativeContract
+    {
+        public override string ServiceName => "Neo.Native.Oracle.Policy";
+
+        public override int Id => -4;
+
+        private const byte Prefix_Validator = 24;
```

Comment:
We need a general place for these `Prefix`.

Reply:
Does Prefix have a clear counting rule?I think the value of Prefix is ​​a bit confusing, although there will not be a big problem because of the existence of ID. If possible, we can specify a counting rule. @erikzhang